dictionaries/software-terms/src/coding-terms.txt

```diff
@@ -80,18 +103,44 @@ commitid
 compat
 concat
 cond
+d0d0caca # ("doo-doo caca") https://en.wikipedia.org/wiki/Hexspeak
+dabbad00 # ("dabba doo") https://en.wikipedia.org/wiki/Hexspeak
 datetime
-deadbeef
+dead10cc # ("dead lock") https://en.wikipedia.org/wiki/Hexspeak
+dead2bad # ("dead too bad") https://en.wikipedia.org/wiki/Hexspeak
+deadbaad # ("dead bad") https://en.wikipedia.org/wiki/Hexspeak
+deadbabe # ("dead babe") https://en.wikipedia.org/wiki/Hexspeak
+deadbeaf # ("dead beaf") https://en.wikipedia.org/wiki/Hexspeak
+deadbeef # ("dead beef") https://en.wikipedia.org/wiki/Hexspeak
+deadc0de # ("dead code") https://en.wikipedia.org/wiki/Hexspeak
+deadd00d # ("dead dude") https://en.wikipedia.org/wiki/Hexspeak
+deaddead # ("dead dead") https://en.wikipedia.org/wiki/Hexspeak
+deadfa11 # ("dead fall" or "dead fail") https://en.wikipedia.org/wiki/Hexspeak
+deadfeed # ("dead feed") https://en.wikipedia.org/wiki/Hexspeak
+decafbad # ("decaf bad") https://en.wikipedia.org/wiki/Hexspeak
+defec8ed # ("defecated") https://en.wikipedia.org/wiki/Hexspeak
 dname
 dosomething
 dotfile
 doublecircle
 doublequote
 dword
+e011cfd0 # ("docfile0") https://en.wikipedia.org/wiki/Hexspeak
 endtime
 eqeqeq
```

Comment:
Is this one a command or something that can be found in a test

Reply:
And this one @jason3s

Reply:
I just included the entire list from Hexspeak. It doesn't hurt anything.

Reply:
eqeqeq was there before, my point was to add a comment with it

Reply:
Ahh, I missed that.

Reply:
I'm eagle-eyes Christophe 🦅